lib/actions/push.ts

```diff
@@ -367,14 +364,12 @@ export const push: CommandDefinition<
 								: options.env || [],
 						convertEol: options['convert-eol'] || false,
 					}),
-				)
-					.catch(BuildError, e => {
-						exitWithExpectedError(e.toString());
-					})
-					.nodeify(done);
+				).catch(BuildError, e => {
+					throw new ExpectedError(e.toString());
+				});
 				break;
 			default:
-				exitWithExpectedError(
+				throw new ExpectedError(
 					stripIndent`
 					Build target not recognised. Please provide either an application name or device address.
```

Comment:
I know this hasn't been changed. But it we are using US English then this should be spelt `recognized`.

Reply:
I will Americani**S**e it :-)